src/index.js

```diff
@@ -418,6 +421,7 @@ function Bigtable(options) {
   this.api = {};
   this.auth = googleAuth(options);
   this.projectId = options.projectId || '{{projectId}}';
+  this.appProfileId = options.appProfileId || null;
```

Comment:
Just `this.appProfileId = options.appProfileId` is fine. If the user didn't provide it, it will resolve it to `undefined`, which will be handled the same as `null`.

Reply:
ok, updating that